frontend/src/pull-card/flags.tsx

```diff
@@ -62,6 +62,7 @@ function PullFlag({variant, title, href, icon}: PullFlagProps) {
    return (href ?
    <Link
       sx={styles}
+      isExternal={true}
```

Comment:
Is setting `isExternal` equal to `true` for explicitness? I see from the docs we can just declare the prop -- https://chakra-ui.com/docs/components/navigation/link

![External Link](https://user-images.githubusercontent.com/22064420/162477020-a2b4dbd9-021c-4040-bf07-d856078ac49b.png)

Reply:
TIL, OMG, I didn't know this!